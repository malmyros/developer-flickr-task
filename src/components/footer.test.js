import React from 'react';
import { shallow } from 'enzyme';
import Footer from './footer';

describe('footer', () => {
  it('should render without crashing', () => {
    const component = shallow(<Footer/>);
    expect(component).toMatchSnapshot();
  });
});
