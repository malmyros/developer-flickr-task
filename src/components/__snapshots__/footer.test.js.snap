// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer should render without crashing 1`] = `
<footer
  className="text-muted pt-3 pb-3"
>
  <div
    className="container"
  >
    <p
      className="float-right"
    >
      <a
        href="/#"
        title="back to top"
      >
        Back to top
      </a>
    </p>
    <p>
      Â© 
      PhotoStreamer
       is a basic example of representing competence in building single page applications with 
      <strong>
        react
      </strong>
      <br />
      while consuming 
      <strong>
        Flickr
        's
      </strong>
       RESTful API.
    </p>
  </div>
</footer>
`;
